### PR TITLE
Quality of live: Improved error messages

### PR DESCRIPTION
**Pull request by romanc**
Small quality of live PR that improves two error messages. Especially if the state was called `state` (as per default), it was hard to parse at first sight

image

For some reason, `yapf` was also changing the formatting of other code parts. I thus made two commits

1. Format the two files that I was about to edit.
2. Improve the error messages in the freshly formatted files.

This allows to separate real changes from purely stylistic ones.

Would it be an idea to enforce linting with a GitHub Actions workflow? I'm happy to write an issue about this (and contribute early next year).

### CODE REVIEW

**Comment by romanc** [comment]
Not sure why GitHub Actions wasn't able to fetch the pace data. Works on my machine. Maybe just a network glitch. @tbennun would you mind re-running that workflow?

/cc @FlorianDeconinck FYI

**Comment by phschaad** [comment]
This has happened quite a few times the last 2 days. An occasional re-run fixes the issue, but there may be an underlying issue.

**Comment by romanc** [comment]
> This has happened quite a few times the last 2 days. An occasional re-run fixes the issue, but there may be an underlying issue. I've restarted the workflow

Hmm weird. Let me propose two things

1.`wget` would normally retry (as far as I know), but not with fatal errors (like the 503 that we've seen in this case). There's `--retry-connrefused` that we can leverage to even retry under these circumstances. I'll setup a quick PR for this since you say that this occurs often.
2. This data rarely changes. We could leverage GHA caches to avoid re-downloading the files and instead fetching them for a GHA cache. I'll have to discuss with @FlorianDeconinck time-budget wise. Shouldn't take too long, but is more than just adding a retry flag.

**Comment by phschaad** [comment]
This seems reasonable, good idea. Thanks! Using GHA caches could make this more robust

**Review by phschaad — APPROVED** [review]
These changes LGTM, thank you.

As for the point about running a formatter through GHA when PRs are merged: I will bring this point up in our weekly DaCe meeting to discuss, thank you for the suggestion.

**Comment by romanc** [comment]
> As for the point about running a formatter through GHA when PRs are merged: I will bring this point up in our weekly DaCe meeting to discuss, thank you for the suggestion.

Thanks for bringing this to the weekly meeting.
